curriculum/challenges/english/01-responsive-web-design/applied-visual-design/create-a-more-complex-shape-using-css-and-html.md

```diff
@@ -39,7 +39,7 @@ The `background-color` property of the `heart::after` selector should be pink.
 
 ```js
 assert(
-  code.match(/\.heart::after\s*?{\s*?background-color\s*?:\s*?pink\s*?;/gi)
+  code.match(/\.heart::after\s*?{[^\}]*?background-color\s*?:\s*?pink\s*?;/gi)
```

Comment:
```suggestion
  /({|;)\s*background-color\s*:\s*pink\s*(;|})/g.test(heartAfter)
```

Reply:
@moT01 I had to make a slight tweak to capture that missing semi-colon issue.

Reply:
I suppose it helps. It doesn't stop the semi-colon missing on other properties though, just the one right before it if I am understanding correctly. It might be a bit overkill, but I'm wondering if we could run the css through a validator or something.

I'm fine with the suggestion, it's an edge case - might as well catch it if we can.

Reply:
True, though a missing semi-colon on another property won't necessarily break the rendering of this property.